src/components/LayoutHeader/LanguageDropdown.js

```diff
@@ -2,7 +2,6 @@
  * Copyright (c) 2013-present, Facebook, Inc.
  *
  * @emails react-core
- * @flow
```

Comment:
I assume this is why Circle-CI is failing. Re-adding @flow here and fixing the errors (npm run flow) will probably make Circle-CI pass. I would fix them myself as they are just code format errors but I'm not sure if I can edit somebody else's branch?

Reply:
Done but it doesn't seem to be re-running